submissions/vakulinina/main.css

```diff
@@ -0,0 +1,208 @@
+* {
+  margin: 0;
+  padding: 0;
+  appearance: none;
+  list-style: none;
+  text-align: center;
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  height: 50px;
+  justify-content: flex-end;
+  align-items: center;
+  margin: 20px 0;
+  font-family: Arial, sans-serif;
+  font-size: 24px;
+  color: #757575;
+}
+
+.navbar li {
+  position: relative;
+  margin: 5px 20px;
+}
+
+.navbar a {
+  display: block;
+  padding: 5px;
+  color: #757575;
+}
+
+.navbar > li > a:focus,
+.navbar > li > input:focus {
+  background-color: rgb(224, 224, 224, 0.5);
+}
+
+.navbar > li > a:hover {
+  background-color: rgb(224, 224, 224, 0.7);
+  border-radius: 4px;
+}
+
+.app-button-label {
+  display: block;
+  width: 40px;
+  border-radius: 4px;
+}
+
+label:hover {
+  cursor: pointer;
+}
+
+.app-button {
+  display: block;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  z-index: -1;
+}
+
+.avatar {
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+}
+
+label img {
+  width: 100%;
+  vertical-align: middle;
+}
+
+a img {
+  vertical-align: middle;
+}
+
+a:visited {
+  color: #757575;
+}
+
+/* App-menu */
+
+.app-menu {
+  display: none;
+}
+
+.app-menu::-webkit-scrollbar  {
+  width: 3px;
+}
+
+.app-menu::-webkit-scrollbar-thumb  {
+  background-color: rgba(133, 132, 132, 0.5);
+  border-radius: 5px;
+}
+
+.app-menu .menu-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  box-sizing: border-box;
+  width: 30%;
+  max-height: 24%;
+  margin: 0;
+  border-radius: 4px;
+}
+
+.app-menu .menu-item a {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+  border: 2px solid transparent;
+  border-radius: 4px;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  padding: 20%;
+}
+
+.app-menu .menu-item img {
+  max-width: 90%;
+  max-height: 60%;
+  margin-bottom: 10%;
+}
+
+.app-menu .menu-item a:focus {
+  border-color: rgba(224, 224, 224, 0.7);
+  background-color: transparent;
+  outline: none;
+}
+
+.app-menu .menu-item:hover {
+  background-color: rgba(224, 224, 224, 0.7);
+}
+
+.app-button:checked ~ label {
+  background-color: rgba(224, 224, 224, 0.5);
+}
+
+.app-button:checked ~ .app-menu {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-around;
+  align-content: flex-start;
+  overflow: auto;
+  position: absolute;
+  box-sizing: border-box;
+  top: 60px;
+  right: -90px;
+  width: 400px;
+  max-width: 400px;
+  height: 600px;
+  padding: 10%;
+  box-shadow: 0 0 20px gray;
+  border-radius: 4px;
+  font-size: 0.7em;
+  line-height: 1.5em;
+}
+
+.app-menu .menu-item:nth-child(n + 13),
+.app-menu .div-line {
+  display: none;
+}
+
+.div-line {
```

Comment:
It's better to change name of this class.

Reply:
Thank you, all done.

Reply:
Replaced it with pseudo-element in css